__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require('supertest');
+const testData = require("../db/data/test-data");
+const db = require('../db/connection');
+const seed = require("../db/seeds/seed");
+const app = require('../app');
+
+
+
+afterAll(() => {
+    return db.end();
+  });
+  
+  beforeEach(() => {
+    return seed(testData);
+  });
+
+  describe('GET/api/topics',() => {
+    test('returns status 200 and an array', () => {
+      return request(app).get("/api/topics")
+      .then((response) => {
+        const topics = response.body.topic;
+        expect(response.status).toBe(200)
+        expect(Array.isArray(topics)).toBe(true)
+      })
+    })
+    test('Results Should contain a property of slug and description', () => {
+      return request(app).get("/api/topics")
+      .then((response) => {
+        const topics = response.body.topic;
+        expect(response.status).toBe(200)
+        expect(topics.every((topic) => {
```

Comment:
.every is a wonderful way to check that everything in the array has these properties, however you should also be checking that the length is higher than 0. It will always return true regardless of what the predicate you provide it with is doing if the length if zero :)